examples/purchaseTesterTypescript/ios/File.swift

```diff
@@ -0,0 +1,6 @@
+//
+//  File.swift
+//  PurchaseTester
+//
+
+import Foundation
```

Comment:
Was this added by RN? 😅

Reply:
I guess this might be the usual "single, empty swift file to enable swift support" thing? 
If so, let's call it `Dummy.swift` to match other repos where we've done the same thing

Reply:
yeah, this was created by the new plugin creation tool, I can rename it to dummy

Reply:
this file is not part of the project in the created empty plugin, so not sure it actually does anything, but I will leave it just in case